.gitlab-ci.yml

```diff
@@ -13,7 +13,7 @@ stages:
     VERSION: ${CI_COMMIT_REF_SLUG}
     OSTAG: <Tag suffix corresponding to version>
     # Use single quotes to escape colon
-    DESTINATION: '${CI_REGISTRY_IMAGE}/${IMAGE}:${VERSION}-${OSTAG}'
+    DESTINATION: 'ghcr.io/$CI_REGISTRY_USER/${IMAGE}:${VERSION}-${OSTAG}'
   before_script:
     # Prepare Kaniko configuration file
     - echo "{\"auths\":{\"$CI_REGISTRY\":{\"username\":\"$CI_REGISTRY_USER\",\"password\":\"$CI_REGISTRY_PASSWORD\"}}}" > /kaniko/.docker/config.json
```

Comment:
Looking at this again, I think this might also need adjustment to point things towards the github registry instead of the gitlab one.

I suppose we would need to override 
- `CI_REGISTRY` to point to `ghcr.io/MuonColliderSoft`
- `CI_REGISTRY_USER` would probably need to be changed to a github user with permissions to push to the github registry
- `CI_REGISTRY_PASSWORD` probably needs to be a github token that we store via the gitlab secrets, such that we can then inject it here again.